File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_nth_pset.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Session var_0;
    MPI_Info var_1;
    int var_2;
    int *var_3;
    char *var_4;
    int ret;
    /* calls */
    ret = MPI_Session_get_nth_pset(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Session_get_nth_pset(var_0, var_1, var_2, var_3, var_4);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_nth_pset.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       CHARACTER*(10) var_4
       INTEGER var_5
        call mpi_session_get_nth_pset(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_session_get_nth_pset(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    